indico/modules/designer/templates/template.html

```diff
@@ -14,6 +16,38 @@
 {% endblock %}
 
 {% block content %}
+    {% if tpls_count %}
+        <div class="affected-targets-warning weak-hidden">
+            {% call message_box('warning', fixed_width=true) %}
+                <p>
+                    {% set link_start %}<a href="#" class="js-affected-objects">{% endset %}
+                    {% set link_end %}</a>{% endset %}
+                    {%- trans count=tpls_count, link_start=link_start|safe, link_end=link_end|safe -%}
```

Comment:
The last thing you mentioned doesn't work, I guess I do need to specify the `count=tpls_count`
```
TemplateAssertionError: unknown variable 'tpls_count' for pluralization
```